packages/block-library/src/gallery/block.json

```diff
@@ -48,6 +48,11 @@
 		"columns": {
 			"type": "number"
 		},
+		"caption": {
+			"type": "string",
+			"source": "html",
+			"selector": "figcaption"
```

Comment:
I think this selector could select the `figcaption` of one of the individual images. Adding a classname to differentiate it might help.

Reply:
Done!